tesseract/tesseract_collision/include/tesseract_collision/core/common.h

```diff
@@ -116,8 +124,11 @@ inline ContactResult* processResult(ContactTestData& cdata,
     ContactResultVector data;
     if (cdata.req.type == ContactTestType::FIRST)
     {
-      data.emplace_back(contact);
-      cdata.done = true;
+      if (contact.distance > cdata.safety_margin_data.getPairSafetyMarginData(key.first, key.second))
```

Comment:
Is this correct? I'm not sure if these keys align.

Reply:
Where is the definition of SafetyMarginData?

Reply:
It should be but I think we want to update the lookup table to work how the allowed collision matrix works.

Reply:
It is defined in types.h

Reply:
done